StudCity.Frontend/src/components/SearchUsersForCreatedChats/index.jsx

```diff
@@ -0,0 +1,172 @@
+			setUsers(data.items)
+			setTotalItemsCount(data.pageViewModel.totalItemsCount)
+		}
+		if (data?.items?.length === 0) {
+			setUsers([])
+		}
+	}, [data.items])
+
+	useEffect(() => {
+		dispatch(getUsersSearch({
+			page: 1,
+			searchWord: searchData
+		}))
+	}, [searchData])
+
+	useEffect(() => {
+
+		if (indexPage > 1 && (totalItemsCount / (10 * (indexPage - 1))) > 1) {
+			dispatch(getUsersSearch({
+				page: indexPage,
+				searchWord: searchData
+			}))
+		}
+	}, [indexPage])
+
+	const handleSearch = ({ target }) => {
+		setSearchData(target.value);
+		setUsers([])
+		setIndexPage(1)
+	}
+
+	const handleScroll = () => {
+		if (containerRef.current) {
+			if ((containerRef.current.clientHeight + containerRef.current.scrollTop > containerRef.current.scrollHeight - 1) && !loading) {
+				setIndexPage((prev) => prev + 1);
+			}
+		}
+	};
+
+	const handleClick = (user) => {
+		setSelectedUsers(prev => {
+			if (prev.some(item => item.id === user.id)) {
+				return prev.filter(elem => elem.id !== user.id)
+			}
+			return [...prev, user]
+		})
+	}
+
+	return (
+		<div className="h-full w-full mx-auto flex flex-col">
+			<div className="w-full mx-auto text-xl">
+				<Tabs
+					active={tabIndex}
+					setActive={setTabIndex}
+					labels={LABELS_USERS_GROUPS}>
+				</Tabs>
+			</div>
+			<div className="h-5/6 w-full">
+				<div className="h-1/6 flex">
+					<TextField
+						className="w-9/12 my-2 h-5/6"
+						placeholder='Search'
+						onChange={handleSearch}
+					/>
+					<div className="w-2/12 ml-auto mr-0">
+						<SelectedUsers selectedUsers={selectedUsers} />
+					</div>
+				</div>
+				<div className="h-[320px] max-h-5/6 w-full flex pt-2" >
+					<Scroller parentRef={containerRef}>
+						{indexPage === 1 && loading
+							? <Spinner />
+							: (
+								users?.length > 0
+									? <div className="h-fit">
+										{users?.map((elem) => (
+											<User
+												user={elem}
+												key={elem.id}
+												onClick={handleClick}
+												className={selectedUsers?.some(item => item.id === elem.id) ? 'bg-customGreen' : ''}
+											/>
+										))}
+										{indexPage > 1 && loading && (<div className="h-24 flex w-full">
+											<Spinner />
+										</div>)}
+									</div>
+									: <div className="w-full h-full flex"><p className="m-auto">Nothing found</p></div>
+							)
+						}
+					</Scroller>
+				</div>
+			</div>
+		</div>
+	)
+}
+
+function User({ user, onClick, className }) {
+	return (
+		<div
+			className={`w-[97%] border rounded-md my-2 p-3 cursor-pointer h-20 flex last:my-0 last:mt-2 first:mt-2 hover:bg-customGreen duration-200 ${className}`}
+			onClick={() => onClick(user)}
+		>
+			<div className="flex items-center">
+				<Avatar className="w-16 h-16" src={user?.avatar} />
+			</div>
+			<div className="ml-5 flex flex-col justify-center">
+				<p className="font-bold">{user?.fullName}</p>
+				<p className="text-gray-500">{user?.lastSeen || "last seen 5 minutes ago"}</p>
```

Comment:
is this desired logic? if we don't have info about when user was online we just show that he was online 5 minutes ago, maybe we should use something like in Telegram, like 'last seen recently', what do you think @Vova-Romaniuk @maryanchuk04

Reply:
there will be other info here, this test info

Reply:
@Serhiy-dotcom I think so let it be, and in the future we will bring it out, because I plan to bring out all the inscriptions in the translation file